### ISSUE TITLE
extracting Infobox data from Turkish wikipages

### ISSUE THREAD

**Issue by developmentortayfun** [spencermountain/wtf_wikipedia#6]
Hi, 

I'm trying to extracting information by using your script; however, I couldn't be able to do that. You can see the rawdata of Toronto written in Turkish below. 

http://en.wikipedia.org/w/index.php?action=raw&title=toronto
http://tr.wikipedia.org/w/index.php?action=raw&title=toronto

I'm trying to list matching definitions:
disambig words: "anlam ayrımı"
infobox: "bilgi kutusu" (in Toronto case, "yerleşim bilgi kutusu" which means "Infobox settlement" )
category:"kategori"
redirect: "yönlendirme"

No success! Could you please help?

Thanks..

**Comment by spencermountain** [comment]
hey, thanks for doing this. It was two bugs that were my fault.
I've added your terms, but i'm worried about the left-to-right "... bilgi kutusu" infobox statements, which I will later try to fix properly.

```javascript
wtf_wikipedia.from_api("Toronto", 'tr', function(s){
  console.log(wtf_wikipedia.parse(s)) 
})
```
cheers!

**Comment by developmentortayfun** [comment]
Thanks a lot bro! it saved my life. I've created several tool for semantic info scraping and I think I can create another one for infoboxes. Then, I've started to create my own parser for that purpose; however, as you said it is never a good idea :) I've made some modifications for Turkish language (for urls) in wtf_wikipedia and now, it works great. thanks a lot again for your great effort! I'll mention about you in my phd thesis..

Best..
